### ISSUE TITLE
Matchmaking fails after three seconds

### ISSUE THREAD

**Issue by alanmur** [aws-samples/amazon-gamelift-ultrafrogroyale-large-match-sample#3]
Instructions fail to inform user to increase ConnectUltraFrogRoyalClient timeout beyond three seconds, to at least a minute. [Matchmaking rules wait for at least 25 seconds before expanding](https://github.com/aws-samples/amazon-gamelift-ultrafrogroyale-large-match-sample/blob/master/AWS/UltraFrogRoyaleFreeForAll-debug_MatchmakingRuleSet.json#L36) to the point where a 3-player match can be made.

**Comment by tharinduprabath** [comment]
> 
> 
> Instructions fail to inform user to increase ConnectUltraFrogRoyalClient timeout beyond three seconds, to at least a minute. [Matchmaking rules wait for at least 25 seconds before expanding](https://github.com/aws-samples/amazon-gamelift-ultrafrogroyale-large-match-sample/blob/master/AWS/UltraFrogRoyaleFreeForAll-debug_MatchmakingRuleSet.json#L36) to the point where a 3-player match can be made.

Yeah, They forgot that but I figure it out. I'm new to gamelift. It's worth if they can mention it in the instructions. Btw thanks for the great example.
